.github/workflows/build_main_documentation.yml

```diff
@@ -134,7 +136,9 @@ jobs:
             if [[ `git status --porcelain` ]]; then
               git add ${{ inputs.notebook_folder }}
               git commit -m "Updated ${{ inputs.package }} doc notebooks with commit ${{ inputs.commit_sha }} \n\nSee: https://github.com/huggingface/${{ inputs.package }}/commit/${{ inputs.commit_sha }}"
-              git push origin master
+              git push origin main ||
+              (echo "Failed on the first try, rebasing and pushing again" && git pull --rebase && git push origin master) ||
+              (echo "Failed on the second try, rebasing and pushing again" && git pull --rebase && git push origin master)
```

Comment:
master or main?

Reply:
Can we rename all those masters to main already ;-)

Reply:
Let's rename `transformers`' `master` branch to `main` on Monday? :smiley:

Reply:
Let's goooooooooooooooooooooooooooo!

Reply:
Necessary changes for the doc moon-landing:
https://github.com/huggingface/moon-landing/pull/2409
https://github.com/huggingface/doc-build/pull/6